guides/plugins/plugins/administration/add-custom-component.md

```diff
@@ -1,4 +1,4 @@
-# Add own component
+# Add own component 
```

Comment:
Just curious: Is this a problem when displayed in Gitbook ? 
Should we check for this in future PRs ?

Reply:
It was a recommendation by @elkmod to update the broken articles and trigger their fix - as now the fix by Gitbook isn't applied yet.

Reply:
@lernhart We just need to trigger any change on those articles. :-)

Reply:
May I remove this silly change afterwards or should we leave it live that? :D